keyboards/mechlovin/infinity875/info.json

```
@@ -4,40 +4,2202 @@
     "url": "",
     "maintainer": "Mechlovin' Studio",
     "usb": {
-        "vid": "0x4D4C",
-        "pid": "0x0875",
-        "device_version": "0.0.1"
+        "vid": "0x4D4C"
     },
-    "rgb_matrix": {
-        "driver": "ws2812"
-    },
-    "indicators": {
-        "caps_lock": "D6",
-        "scroll_lock": "B4"
-    },
-    "ws2812": {
-        "pin": "E2"
-    },
-    "processor": "atmega32u4",
-    "bootloader": "atmel-dfu",
     "layouts": {
```

add Community Layout support
```suggestion
    "community_layouts": {
        "tkl_ansi",
        "tkl_ansi_split_bs_rshift",
        "tkl_ansi_tsangan",
        "tkl_ansi_tsangan_split_bs_rshift",
        "tkl_f13_ansi",
        "tkl_f13_ansi_split_bs_rshift",
        "tkl_f13_ansi_tsangan",
        "tkl_f13_ansi_tsangan_split_bs_rshift",
        "tkl_f13_iso",
        "tkl_f13_iso_split_bs_rshift",
        "tkl_f13_iso_tsangan",
        "tkl_f13_iso_tsangan_split_bs_rshift",
        "tkl_iso",
        "tkl_iso_split_bs_rshift",
        "tkl_iso_tsangan",
        "tkl_iso_tsangan_split_bs_rshift"
    },
    "layouts": {
```